example/server/liquid_servlet.rb

```diff
@@ -1,23 +1,25 @@
+# frozen_string_literal: true
+
 class LiquidServlet < WEBrick::HTTPServlet::AbstractServlet
-  def do_GET(req, res)
+  def do_get(req, res)
```

Comment:
Is this change ok? was this not overwriting a superclass method?

Reply:
I'm looking in to this one at the moment.

Reply:
To get around rubocop I've renamed these methods to just `get` and `post` and then aliased them to the original `do_GET` and `do_POST` respectively